timescaledb/docker-compose.timescaledb.yml

```diff
@@ -93,89 +65,62 @@ services:
         # - 'connector-server' - for AcraServer and AcraConnector interconnection
         networks:
             - server-timescaledb
-            - connector-server
+            - metricsource-server
+            - grafana-server
+            - prometheus-server
+        ports:
+            - "9393:9393"
```

Comment:
Do we need to open Acra ports to the host? I thought it is only for internal usage

Reply:
previously we exported acra-connector's ports to allow users connect via `psql` from host machine. there are same reason, to allow connect to acra not only by pgadmin, but by any other tool that can work with db: psql, DataGrid, etc

Reply:
Got it, make sense